src/run.js

```diff
@@ -32,9 +34,7 @@ module.exports = function run(regions, inputDir, outputDir) {
     inputFiles,
     processFile.bind(null, regions, inputDir, outputDir),
     function (code) {
-      if (code) {
-        process.exit(code);
-      }
+      process.exit(code || 0);
```

Comment:
yeah, the 0's are probably just making it awkward to read. I'll take them out in both places.